src/core/listener.c

```diff
@@ -757,45 +758,49 @@ QuicListenerParamSet(
         const void* Buffer
     )
 {
-    if (Param == QUIC_PARAM_LISTENER_CIBIR_ID) {
```

Comment:
Why did you change all this code? AFAICT, you didn't do anything but change from if/else to switch.

Reply:
Refactoring to use the switch statement. Let me know if you want to reverse this refactor.

Reply:
Since it's not directly required for this change, let's revert it.

Reply:
Reverted.